internal/check_api_change.sh

```diff
@@ -47,6 +47,7 @@ trap cleanup EXIT
 # Move to a temporary directory while installing apidiff to avoid changing
 # the local .mod file.
 ( cd "$INSTALL_DIR" && exec go mod init unused )
+( cd "$INSTALL_DIR" && exec go get golang.org/x/exp/cmd/apidiff )
 ( cd "$INSTALL_DIR" && exec go install golang.org/x/exp/cmd/apidiff )
```

Comment:
IIUC in Go 1.16 there is a better way to do this, according to https://golang.org/doc/go1.16#go-command (untested):

```suggestion
go install golang.org/x/exp/cmd/apidiff@latest
```

Reply:
Done.